app/server/routes/v1/post/post.js

```diff
@@ -134,6 +135,13 @@ router.post("/", async (req, res) => {
     );
     req.log.addAction("Community engagement updated.");
 
+    req.log.addAction("Updating community latest activity.");
+    await Community.updateOne(
+      { title: post.community },
+      { $set: { latestActivity: moment().format("MMMM Do YYYY, h:mm:ss a") } }
+    );
+    req.log.addAction("Community latest activity updated.");
```

Comment:
do we need to include the community name and timestamp here?

Reply:
Do you mean in the log? Technically the timestamp of the request will be logged, and the community title is also logged as it is part of the request body. Do you think it should be included in the log action like `"Community (Welcome) latest activity updated."`?

Reply:
I think more detail would be nifty imo.